paddle/cinn/auto_schedule/analysis/analyze_ir.h

```diff
@@ -48,5 +48,15 @@ ir::LoweredFunc UpdateFuncWithNewBody(const common::Target& target,
                                       const ir::LoweredFunc& old_func,
                                       ir::Expr& body);  // NOLINT
 
+/**
+ * Get loop var names of reduce axis
+ */
+std::unordered_set<std::string> GetReduceLoopVarName(const ir::Expr block);
```

Comment:
How about `GetReduceLoopVarNames`? The `s` to indicate it returns collection of names.

Reply:
done